### DISCUSSION TITLE
[Unstable] Launching any individual outpost module in the submarine editor's test mode generates a PvP outpost instead, if the module's location type is set to Any

### DISCUSSION THREAD

**Discussion by NotWendy** [FakeFishGames/Barotrauma#14833] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I'm reporting two different issues within this ticket.

- Clown_G3, Clown_G4, Husk_G1, and Husk_G2 outpost modules got imported with their Location Types set to "Any". Should be "Military, Mine, Outpost, Research". That's my bad for not double-checking them in the PR.
- If an individual outpost module with location type "Any" is launched in the submarine editor's test mode, the game gets confused and generates a PvP outpost instead. I don't think that's supposed to happen.

### Reproduction steps

1. Load any outpost module with its Location Types set to "Any" in the submarine editor.
2. Press the Test button.
3. A full PvP outpost is generated, instead of the module you were trying to test.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Should be fix now, and the fix is included in the latest unstable build.
